src/main/net/polydawn/mdm/MdmArgumentParser.java

```diff
@@ -63,6 +63,12 @@ public void onAttach(Argument arg) {}
 		Subparser parser_status = subparsers
 			.addParser("status")
 			.help("list dependencies managed by mdm, and their current status.");
+		parser_status
+			.addArgument("--name")
+			.help("get the status of a particular dependency by name.  If specified, only that dependency will be included in the report.");
+		parser_status
+			.addArgument("--format")
```

Comment:
Ok, I guess I can see why you used format. I'm still not sure that it's exactly the right term, but it seems fine.